index.d.ts

```diff
@@ -0,0 +1,130 @@
+declare module 'fluture' {
+  interface ResolveFunction<T> {
+    (value: T): void
+  }
+  interface RejectFunction {
+    (error: Error): void
+  }
+
+  interface FunctionReturning<T extends Function> {
+    (...t: any[]): T
+  }
+  abstract class Future<T> {
+    done<T> (fn: (err: Error, value: T) => void): Future<T>
```

Comment:
1. We better specify `fork` over `done`: `done` is really just an alias.
1. I'm not sure, but this seems to state that `done` returns a Future, instead it returns `Cancel`. Maybe we can define a type for `Cancel`, as is [done in the README](https://github.com/fluture-js/Fluture#types).